.travis.yml

```diff
@@ -5,7 +5,8 @@ language: ruby
 cache: bundler
 before_install:
   # https://github.com/travis-ci/travis-ci/issues/9383#issuecomment-377680108
-  - gem install bundler
+  # not needed for RubyGems 2.7.7...
+  #- gem install bundler
```

Comment:
Maybe we can change it to `gem update --system 2.7.7` so that further updates of `rubygems` don't break the build?

Reply:
Not sure when 3.x will be released, but ok with that! Hopefully new releases won't break anything... :)